exasol_integration_test_docker_environment/lib/test_environment/analyze_test_container.py

```diff
@@ -11,6 +14,7 @@
 
 
 class AnalyzeTestContainer(DockerAnalyzeImageTask):
+    test_container_content = JsonPickleParameter(TestContainerContentDescription, significant=False)
 
```

Comment:
with significant=False, their can be always only one task of this class.

Reply:
Done for all occurences of 
```
test_container_content = JsonPickleParameter(TestContainerContentDescription, significant=False)
```